gunicorn/util.py

```diff
@@ -551,7 +566,19 @@ def split_request_uri(uri):
         # relative uri while the RFC says we should consider it as abs_path
         # http://www.w3.org/Protocols/rfc2616/rfc2616-sec5.html#sec5.1.2
         # We use temporary dot prefix to workaround this behaviour
-        parts = _compat.urlsplit("." + uri)
+        parts = urlsplit("." + uri)
         return parts._replace(path=parts.path[1:])
 
-    return _compat.urlsplit(uri)
+    return urlsplit(uri)
+
+
+def reraise(tp, value, tb=None):
```

Comment:
Nit: Can you add a comment to note that we borrow this code from six?

Reply:
✅